source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/detail/activitiesList/ManagementActivitiesListView.tsx

```diff
@@ -0,0 +1,67 @@
+import { orderBy } from 'lodash';
+import React from 'react';
+
+import LoadingBackdrop from '@/components/common/LoadingBackdrop';
+import { Section } from '@/components/common/Section/Section';
+import { TableSort } from '@/components/Table/TableSort';
+import { Api_PropertyManagementActivity } from '@/models/api/Property';
+
+import ManagementActivitiesList from './ManagementActivitiesList';
+import { PropertyActivityRow } from './models/PropertyActivityRow';
+
+export interface IManagementActivitiesListViewProps {
+  isLoading: boolean;
+  propertyActivities: PropertyActivityRow[];
+  onDelete: (managementActivityId: number) => void;
+}
+
+const ManagementActivitiesListView: React.FunctionComponent<IManagementActivitiesListViewProps> = ({
+  isLoading,
+  propertyActivities,
+  onDelete,
+}) => {
+  const [sort, setSort] = React.useState<TableSort<Api_PropertyManagementActivity>>({});
+
+  const mapSortField = (sortField: string) => {
+    if (sortField === 'activityType') {
+      return 'activityType.description';
+    } else if (sortField === 'activitySubType') {
+      return 'activitySubType.description';
+    } else if (sortField === 'activityStatusType') {
+      return 'activityStatusType.description';
+    }
+    return sortField;
+  };
+
+  const sortedActivities = React.useMemo(() => {
+    if (propertyActivities?.length > 0) {
+      let items: PropertyActivityRow[] = [...propertyActivities];
+
+      if (sort) {
+        const sortFields = Object.keys(sort);
+        if (sortFields?.length > 0) {
+          const keyName = (sort as any)[sortFields[0]];
+          return orderBy(items, mapSortField(sortFields[0]), keyName);
+        } else {
+          items = orderBy(items, ['activityType'], ['asc']);
+        }
+      }
+      return items;
+    }
+    return [];
+  }, [propertyActivities, sort]);
+
+  return (
+    <Section isCollapsable initiallyExpanded header="Activities List">
+      <LoadingBackdrop show={isLoading} />
```

Comment:
I don't think you need this? there should already be a spinner on the table right?

Reply:
I followed the patter as the Contact list, but you are correct. this was updated to pass the flag to the table.